apps/provisioning_api/lib/Controller/GroupsController.php

```diff
@@ -94,6 +93,8 @@ public function getGroups(string $search = '', ?int $limit = null, int $offset =
 	 *
 	 * 200: Groups details returned
 	 */
+	#[NoAdminRequired]
+	#[AuthorizedAdminSetting(Sharing::class)]
```

Comment:
The common pattern is to name the parameter 

```suggestion
	#[AuthorizedAdminSetting(setting: Sharing::class)]
```